vendor/data/codelists/IATI/2_03/activity/aid_type.yml

```diff
@@ -4,37 +4,6 @@ attributes:
   name: AidType
   complete: '1'
 data:
-  - category: A
```

Comment:
These codelists are maintained by IATI and are occasionally updated - probably not for this PR, but I'm wondering if there's any value in these `.yml` files keeping the complete data set, and then us using some other method to hide certain options from the user?

Whilst not applicable in this case, I've seen errors caused by the database containing values that *previously* were valid and are no longer...

Reply:
I don't know! I would appreciate some guidance about the best way to approach this within the time constraints. @mec ?

Reply:
I think we should keep anything in the `vendor` directory intact. 

If you are looking for an approach, perhaps we could set a const that stores the aid types BEIS want to use and use that to produce the list shown to the user in the UI, this along with a helper that can add the custom descriptions (from the locales we already have for aid types) feels like it would work?